hack/verify-api-groups.sh

```diff
@@ -98,7 +98,14 @@ packages_without_install=(
 )
 known_version_files=(
 	"pkg/master/import_known_versions.go"
-	"pkg/client/clientset_generated/internal_clientset/import_known_versions.go"
+# this file doesn't seem to exist
+# we were not actually checking it previously, due to a bug in the script
+#	"pkg/client/clientset_generated/internal_clientset/import_known_versions.go"
+#
+# these fail on missing k8s.io/kubernetes/pkg/apis/auditregistration/install
```

Comment:
NOTE: these were not listed previously, but seem to be the other currently existing `import_known_versions.go`, they are all failing if added.